apps/core/scenes/search/PlacePicker/PlacePickerContent.js

```diff
@@ -61,27 +66,28 @@ function mapLocationsToOption(locations: ?Array<Location>) {
 }
 
 class PlacePickerContent extends React.Component<Props, State> {
-  state = {
-    searchText: '',
-  };
+  constructor(props: Props) {
```

Comment:
Is there any difference to set state in constructor?

Reply:
There is a difference only when you use props. It seems like props aren't used here, so the previous state of the code should be used.

Reply:
I will change it, I was experimenting with state and didn't change it back

Reply:
done